### PR TITLE
Beginner part 4

### PR DESCRIPTION
**Pull request by saveroz**
Containerize application
1. Web server run on docker :heavy_check_mark: 
Screenshot from 2021-07-01 19-56-55

2. Cron run on docker :heavy_check_mark: 
Screenshot from 2021-07-01 19-53-26

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+COPY . .
+
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o engine /webapp/app
+
+CMD ["./engine", "app"]
+
+
+
+
```
Comment (haveiss) [inline Dockerfile:23]: [nit] unused space

File: Makefile [hunk 2]
```diff
@@ -11,7 +11,13 @@ unit-test:
...
 	@ginkgo -r
-	@docker-compose -f docker-compose.test.yaml down
\ No newline at end of file
+	@docker-compose -f docker-compose.test.yaml down
+
+deploy:
+	@docker-compose -f docker-compose.yaml up --build -d
+
+stop:
+	@docker-compose -f docker-compose.yaml down
```
Comment (haveiss) [inline Makefile]: 👍 nice usage of docker-compose

**Review by haveiss — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (saveroz) [reply Makefile]: thanks to you sir :pray:

File: Dockerfile (hunk 1 above)
Comment (saveroz) [reply Dockerfile]: updated

**Pull request merged**